setup.py

```diff
@@ -111,6 +111,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Python 3.10 is not currently enabled in unit tests. This line also needs to be updated to include python 3.10: https://github.com/googleapis/python-aiplatform/blob/main/owlbot.py#L87